defaults/config.js

```diff
@@ -361,11 +361,24 @@ module.exports = {
 		primaryKey: "uid"
 	},
 
-	// Enables extra debugging output. Turn this on if you experience
-	// IRC connection issues and want to file a bug report.
+	// Extra debugging
 	//
-	// @type     boolean
-	// @default  false
+	// @type     object
+	// @default  {}
 	//
-	debug: false,
+	debug: {
+		// Enables extra debugging output provided by irc-framework.
+		//
+		// @type     boolean
+		// @default  false
+		//
+		ircFramework: false,
+
+		// Enables logging raw IRC messages into each server window.
+		//
+		// @type     boolean
+		// @default  false
+		//
+		raw: false,
+	},
```

Comment:
See https://github.com/thelounge/lounge/pull/783#discussion_r91851434, I still think that it should be possible to set `debug` to `false` and that it should be the default. `debug` mode is an exceptional situation and most users will set it to a global `false`, which essentially disables it for all sub-modes.
`true` should enable all debug options.